app/code/Magento/CatalogImportExport/Model/Import/Product/Option.php

```diff
@@ -52,6 +52,8 @@ class Option extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
 
     const COLUMN_ROW_SORT = '_custom_option_row_sort';
 
+    const OPTION_TITLE_PATTERN = '/option_title\=[+-]?([0-9]*.)?[0-9]+/';
```

Comment:
Lets make this constant private for complete backward compatibility
```suggestion
    private const OPTION_TITLE_PATTERN = '/option_title\=[+-]?([0-9]*.)?[0-9]+/';
```